src/Behat/Gherkin/Node/TableNode.php

```diff
@@ -44,6 +44,11 @@ public function __construct(array $table)
         $columnCount = null;
 
         foreach ($this->getRows() as $row) {
+
+            if (!is_array($row)) {
```

Comment:
this condition is now duplicated. You should removed the later one.

Reply:
see #164